sdk/src/main/java/com/mapbox/maps/MapProjectionUtils.kt

```diff
@@ -0,0 +1,31 @@
+package com.mapbox.maps
+
+import androidx.annotation.VisibleForTesting
+import com.mapbox.bindgen.Value
+import com.mapbox.maps.plugin.MapProjection
+import java.lang.RuntimeException
+
+internal object MapProjectionUtils {
+
+  fun MapProjection.toValue() = when (this) {
+    MapProjection.Globe -> Value.valueOf(hashMapOf(NAME_KEY to Value(GLOBE_PROJECTION_NAME)))
```

Comment:
Do you really need a mutable Map? If no we can use `mapOf` instead.

Reply:
AFAIR difference between Kotlin `mapOf` vs `hashMapOf` is that map preservers sorted order.
In any case `valueOf` could take hash map only.